conf.d/done.fish

```diff
@@ -170,12 +170,12 @@ if test -z "$SSH_CLIENT" # not over ssh
                 end
 
             else if type -q notify-send # Linux notify-send
-                set -l urgency $__done_notification_urgency_level
+                set urgency $__done_notification_urgency_level
                 # override user-defined urgency level if non-zero exitstatus
-                if test $exit_status -ne 0
-                    set urgency "--urgency=critical"
```

Comment:
This is necessary to set the urgency to critical when the exit status is not zero.